extensions/arc/src/ui/dashboards/postgres/postgresComputeAndStoragePage.ts

```diff
@@ -151,8 +151,15 @@ export class PostgresComputeAndStoragePage extends DashboardPage {
 							cancellable: false
 						},
 						async (_progress, _token): Promise<void> => {
-							await this._azdataApi.azdata.arc.postgres.server.edit(
-								this._postgresModel.info.name, this.saveArgs);
+							try {
+								await this._azdataApi.azdata.arc.postgres.server.edit(
+									this._postgresModel.info.name, this.saveArgs);
+							} catch (err) {
+								// If an error occurs while editing the instance then re-enable the save button since
+								// the edit wasn't successfully applied
+								this.saveButton!.enabled = true;
```

Comment:
Before the save would be enabled again once the user changed a value. If an edit did not apply because of the cli validation error wouldn't we want it change first?

Reply:
There's other errors that can happen - such as network issues. In which case forcing them to retype in something is just an annoyance.

Better I think to let the user decide what action is appropriate. If it is a validation error then the error message should be clear enough and if they just hit save again it'll just give them the same error.